### PR TITLE
DAT257-45 Changed the appearence of the website

### PR DESCRIPTION
**Pull request by RezaAmanii**
Changes down below were applyed:
  - Removed the side bar and instead we have a nav bar where the page's link are located
  - Added hovering and clicking effect to pages
  - Added Logo and the name of website
  - Added search bar and search button (With no functionality at the moment)
  - Added the paragraph of the country of the day with its flag image
  - Added 6 grid where we can put features in cards
  And some other minor changes

### CODE REVIEW

File: Client/Layout/MainLayout.razor [hunk 1]
```diff
@@ -2,15 +2,41 @@
...
+            <button type="submit">Search</button>            
         </div>
 
+        <div>
+            <p class="paragraph_country">
+                <img src="sweden-flag.png" alt="Sweden" class="flag" />
+                Sweden</p>
+        </div>
+
+        <div class="grid-container">
```
Comment (filipkauffeldt) [inline Client/Layout/MainLayout.razor]: Shouldn't this be a part of the home page instead of the main layout? The main layout is applied to all pages and should probably only consist of the nav bar and other elements that are present on all pages. Do we really want to have this grid on all pages that we could have on the website?

File: Client/Layout/MainLayout.razor [hunk 2]
```diff
@@ -2,15 +2,41 @@
...
-
     <main>
-        <div class = "app_bar">
-            <AppBar/>
+        <div class="header">
+            <h1 class="title">
+                <img src="Logo.png" alt="logo" class="logo" />
+                Contrasting Horizons</h1>
+            <nav>
+                <ul class="nav_links">
```
Comment (filipkauffeldt) [inline Client/Layout/MainLayout.razor]: Why was this removed from a separate component? Shouldn't we have a separate component for the navbar?

File: Client/Layout/MainLayout.razor [hunk 3]
```diff
@@ -2,15 +2,41 @@
...
+            </nav>
+        </div>
+
+        <div class="search-container">
+            <input type="text" placeholder="Search for a country..." name="search" />
+            <button type="submit">Search</button>            
         </div>
 
+        <div>
+            <p class="paragraph_country">
```
Comment (filipkauffeldt) [inline Client/Layout/MainLayout.razor]: We probably not have this on all pages

**Review by filipkauffeldt — CHANGES_REQUESTED** [review]
I do not think these changes should be a part of the main layout. They should probably be split to a separate navbar component that is used in the main layout and a separate file for the homepage

File: Client/Layout/MainLayout.razor (hunk 1 above)
Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: We dont have a home page thats why its there but when the home page is created the code can be moved to home page.

File: Client/Layout/MainLayout.razor (hunk 2 above)
Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: If navbar would be present in all pages so its better it should be shown in the main page as you mentioned in a comment before. The main idea with making something component is that it could be reused but nav bar is the template for all the pages so I think it doesn't need to be a component.

File: Client/Layout/MainLayout.razor (hunk 3 above)
Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: Its just a place holder for the country that the information would be shown. We can move it to a home page where the country of the day ond dashoboard is located. But at the moment we dont have any home page. Main is what we have. Maybe we can bring it up on the next sprint planning.

File: Client/Layout/MainLayout.razor (hunk 1 above)
Comment (Flaskhals68) [reply Client/Layout/MainLayout.razor]: We do have a home page _(Pages/Home.razor)_.
Also, if we want to place any data in the grid, the grid must be **inside** the DashBoard component.

Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: Fixed it. Now everything that doesn't belongs to the MainLayout is moved to Home page.

File: Client/Layout/MainLayout.razor (hunk 2 above)
Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: I converted to navbar to a component.

File: Client/Layout/MainLayout.razor (hunk 3 above)
Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: Fixed it. Removed it from the MainLayout and instead put it in the Home page.

File: Client/Layout/MainLayout.razor [hunk 4]
```diff
@@ -2,15 +2,17 @@
...
-        <div class = "app_bar">
-            <AppBar/>
+        <div class="header">
+            <h1 class="title">
+                <img src="Logo.png" alt="logo" class="logo" />
+                Contrasting Horizons</h1>
+            <NavBar />
         </div>
-
+        <div class="search-container">
```
Comment (filipkauffeldt) [inline Client/Layout/MainLayout.razor]: Will this search be present on all pages or just on the home page?

Comment (RezaAmanii) [reply Client/Layout/MainLayout.razor]: all pages

File: Client/Components/DashBoard.razor [hunk 5]
```diff
@@ -18,13 +18,23 @@
...
+        <div class="grid-container">
             @foreach (var metric in dataMetrics)
             {
-                <div id="@FormatLabel(metric)" class="data-card">
+                <div id="@FormatLabel(metric)" class="grid-item">
                     <ComparisonComponent CountryComparison="@countryComp" CountryOrigin="@countryCompTwo" date="@date" ResourceType="@metric"></ComparisonComponent>
                 </div>
             }
+
+            @* <div class="grid-container">
```
Comment (Flaskhals68) [inline Client/Components/DashBoard.razor]: Remove comment

**Review by Flaskhals68 — APPROVED** [review]

**Review by filipkauffeldt — APPROVED** [review]
